postgresql11/plan.sh

```diff
@@ -0,0 +1,140 @@
+pkg_upstream_url="https://www.postgresql.org/"
+pkg_license=('PostgreSQL')
+pkg_source="https://ftp.postgresql.org/pub/source/v${pkg_version}/postgresql-${pkg_version}.tar.bz2"
+pkg_shasum="90815e812874831e9a4bf6e1136bf73bc2c5a0464ef142e2dfea40cda206db08"
+pkg_dirname="postgresql-${pkg_version}"
+
+pkg_deps=(
+  core/bash
+  core/gawk
+  core/glibc
+  core/grep
+  core/libossp-uuid
+  core/openssl
+  core/perl
+  core/readline
+  core/zlib
+
+  # for postgis
+  core/libxml2
+  core/geos
+  core/proj
+  core/gdal
+)
+
+pkg_build_deps=(
+  core/coreutils
+  core/gcc
+  core/make
+
+  # for postgis
+  core/diffutils
+)
+
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+pkg_exports=(
+  [port]=port
+  [superuser_name]=superuser.name
+  [superuser_password]=superuser.password
+)
+pkg_exposes=(port)
+
+ext_postgis_version=2.5.1
+ext_postgis_source=http://download.osgeo.org/postgis/source/postgis-${ext_postgis_version}.tar.gz
+ext_postgis_filename=postgis-${ext_postgis_version}.tar.gz
+ext_postgis_shasum=fb137056f43aae0e9d475dc5b7934eccce466f86f5ceeb69ec8b5cea26817a91
+
+do_before() {
+  ext_postgis_dirname="postgis-${ext_postgis_version}"
+  ext_postgis_cache_path="$HAB_CACHE_SRC_PATH/${ext_postgis_dirname}"
+}
+
+do_download() {
+  do_default_download
+  download_file $ext_postgis_source $ext_postgis_filename $ext_postgis_shasum
+}
+
+do_verify() {
+  do_default_verify
+  verify_file $ext_postgis_filename $ext_postgis_shasum
+}
+
+do_clean() {
+  do_default_clean
+  rm -rf "$ext_postgis_cache_path"
+}
+
+do_unpack() {
+  do_default_unpack
+  unpack_file $ext_postgis_filename
+}
+
+do_build() {
+    # ld manpage: "If -rpath is not used when linking an ELF
+    # executable, the contents of the environment variable LD_RUN_PATH
+    # will be used if it is defined"
+    ./configure --disable-rpath \
+              --with-openssl \
+              --prefix="$pkg_prefix" \
+              --with-uuid=ossp \
+              --with-includes="$LD_INCLUDE_PATH" \
+              --with-libraries="$LD_LIBRARY_PATH" \
+              --sysconfdir="$pkg_svc_config_path" \
+              --localstatedir="$pkg_svc_var_path"
+    make world
+
+    # PostGIS can't be built until after postgresql is installed to $pkg_prefix
+}
+
+do_install() {
+  make install-world
+
+  # make and install PostGIS extension
+  HAB_LIBRARY_PATH="$(pkg_path_for proj)/lib:${pkg_prefix}/lib"
+  export LIBRARY_PATH="${LIBRARY_PATH}:${HAB_LIBRARY_PATH}"
+  build_line "Added habitat libraries to LIBRARY_PATH: ${HAB_LIBRARY_PATH}"
+
+  export PATH="${PATH}:${pkg_prefix}/bin"
+  build_line "Added postgresql binaries to PATH: ${pkg_prefix}/bin"
+
+  pushd "$ext_postgis_cache_path" > /dev/null
+
+  build_line "Building ${ext_postgis_dirname}"
+  ./configure --prefix="$pkg_prefix"
+  make
+
+  build_line "Installing ${ext_postgis_dirname}"
+  make install
+
+  popd > /dev/null
+}
+
+# Postgresql9X plans source this plan to get build instructions.
```

Comment:
We can probably delete the lines below, as the are specific to the 9x series of pg packages.

Reply:
agreed.  if we ever need that in the future it should probably go into a different file and be sourced. removing.